src/detail-page/components/LearnList.tsx

```diff
@@ -0,0 +1,73 @@
+import Button from '@material-ui/core/Button';
+import React from 'react';
+import styled from 'styled-components';
+import { ProfessionKey } from '../../data/home-page/ITProfessions';
+import { getData } from '../hooks/useLearnList';
+
+interface LearnListProps {
+    professionId: ProfessionKey;
+  }
+
+const StyledDiv = styled('div')`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    align-items: flex-start;
+`;
+
+const StyledButton = styled(Button)`
+    border-radius: 0;
+    font-weight: bold;
+    text-transform: none;
+    border: 2px solid ${props => props.theme.palette.primary.main};
+
+    &:hover {
+        border: 2px solid ${props => props.theme.palette.primary.main};
+    }
+`;
+
+const StyledList = styled('ul')`
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 1rem;
+    padding-left: 1.5rem;
+`;
+
+const StyledLi = styled('li')`
+    margin: .8rem 0;
+    font-size: .95rem;
+    position: relative;
+
+    &::first-letter {
+        text-transform: uppercase;
```

Comment:
Oh... that's weird.
https://caniuse.com/?search=%3A%3Afirst-letter - but actually, They write that it can be something wrong on Firefox. (I didn't check this tab)
So I can change it to JS solution, to be sure that will work for all browsers. :)